client/src/pages/GameView/GameView.js

```diff
@@ -42,37 +43,54 @@ class GameView extends Component {
         played: [],
         excluded: null
       }
-    }
+    },
+    active: "discardDiv"
   };
 
+  handleClick(event) {
+    console.log("wut");
```

Comment:
The console.log() should be commented out with the rest of this function. Do we need this handleClick() function? We could delete it, since it isn't getting used.

Reply:
Good catch, yeah I've already removed that clicker and changed some more things as well